### PR TITLE
Docs: Fix the usage/cms/netlify.md

### PR DESCRIPTION
**Pull request by trkw**
https://github.com/netlify/netlify-cms/issues/142
- Fix the github-api to github
- Don't need that "vendor" asset...

I made some changes. Would you be able to check them?

### CODE REVIEW

File: docs/content/docs/usage/cms/netlify.md [hunk 1]
```diff
@@ -32,14 +32,12 @@ with 2 files :
...
-
+      <meta name="viewport" content="width=device-width,  initial-scale=1.0" />
       <title>Content Manager</title>
-      <link rel="stylesheet" href="//cms.netlify.com/assets/cms.css" />
+      <link rel="stylesheet" href="https://unpkg.com/netlify-cms@^0.1.1/dist/cms.css" />
     </head>
     <body>
-      <script src="//cms.netlify.com/assets/vendor.js"></script>
-      <script src="//cms.netlify.com/assets/cms.js"></script>
+      <script src="https://unpkg.com/netlify-cms@^0.1.1/dist/cms.js"></script>
```
Comment (MoOx) [inline docs/content/docs/usage/cms/netlify.md]: Do we really need the ^0.1.1 in the url? I guess html won't change that much even if there is breaking change somewhere else?

File: docs/content/docs/usage/cms/netlify.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ with 2 files :
       <meta charset="utf-8" />
       <meta name="viewport" content="width=device-width,  initial-scale=1.0" />
       <title>Content Manager</title>
-      <link rel="stylesheet" href="https://unpkg.com/netlify-cms@^0.1.1/dist/cms.css" />
+      <link rel="stylesheet" href="https://unpkg.com/netlify-cms/dist/cms.css" />
```
Comment (trkw) [inline docs/content/docs/usage/cms/netlify.md]: Include the latest version

File: docs/content/docs/usage/cms/netlify.md [hunk 3]
```diff
@@ -34,10 +34,10 @@ with 2 files :
...
     </head>
     <body>
-      <script src="https://unpkg.com/netlify-cms@^0.1.1/dist/cms.js"></script>
+      <script src="https://unpkg.com/netlify-cms/dist/cms.js"></script>
```
Comment (trkw) [inline docs/content/docs/usage/cms/netlify.md]: Include the latest version

File: docs/content/docs/usage/cms/netlify.md (hunk 1 above)
Comment (trkw) [inline docs/content/docs/usage/cms/netlify.md]: ummmm...

Version specified in this page ...

https://github.com/netlify/netlify-cms#installing

**Pull request merged**